src/sc_crawler/vendors/gcp.py

```diff
@@ -1,38 +1,883 @@
+    "Hyperdisk Throughput Capacity": "hyperdisk-throughput",
+    "Hyperdisk Balanced Capacity": "hyperdisk-balanced",
+}
+
+
+def _server_family(server_name: str) -> str:
+    """Look up server family based on server name"""
+    # example server names: f1-micro, n2d-standard-96
+    prefix = server_name.lower().split("-")[0]
+    if prefix in SERVER_FAMILIES:
+        return prefix
+    raise KeyError(f"Not known server family for {server_name}")
+
+
+@cache
+def _skus_dict():
+    """Look up all Compute Engine SKUs and return in a lookup dict."""
+    skus = _skus("Compute Engine")
+    lookup = nesteddefaultdict()
+    for sku in skus:
+        # skip not processed items early
+        if sku.category.resource_family not in ["Compute", "Storage"]:
+            continue
+        if sku.category.resource_family == "Compute":
+            if sku.category.usage_type not in ["OnDemand", "Preemptible"]:
+                continue
+        if sku.category.resource_family == "Storage":
+            if sku.category.usage_type != "OnDemand":
+                continue
+            if sku.category.resource_group not in ["HDD", "SSD", "HDBSP", "HDTSP"]:
+                continue
+
+        # helper variables
+        regions = sku.service_regions
+        if sku.category.usage_type == "OnDemand":
+            allocation = "ondemand"
+        else:
+            allocation = "spot"
+        price_tiers = sku.pricing_info[0].pricing_expression.tiered_rates
+        assert len(price_tiers) == 1
+        price = price_tiers[0].unit_price.nanos / 1e9
+        currency = price_tiers[0].unit_price.currency_code
+
+        if sku.category.resource_family == "Compute":
+            # servers with pricing as-is
+            if sku.category.resource_group in ["F1Micro", "G1Small"]:
+                name = sku.category.resource_group[:2].lower()
+                for region in regions:
+                    lookup["instance"][name][region][allocation] = (price, currency)
+                continue
+
+            # servers with CPU + RAM pricing
+            if (
+                sku.category.resource_group in ["CPU", "RAM"]
+                and "Custom" not in sku.description
+                and "Sole Tenancy" not in sku.description
+                and (
+                    "Instance Core running in" in sku.description
+                    or "Instance Ram running in" in sku.description
+                )
+            ):
+                catgroup = sku.category.resource_group.lower()
+                family = sub(r"^Spot Preemptible ", "", sku.description)
+                family = sub(r" Instance.*", "", family)
+                family = sub(r" AMD$", "", family)
+                family = sub(r" Arm$", "", family)
+
+                # extract instance family from description (?!)
+                for k, v in SERVER_DESCRIPTION_TO_FAMILY.items():
+                    if family == k:
+                        family = v
+                family = family.lower()
```

Comment:
... wouldn't be this enough here?
```suggestion
                family = SERVER_DESCRIPTION_TO_FAMILY.get(family, family).lower()
```
I'm not sure if a miss could be possible, but the above code should replicate the same even in that case.